blogs/2016/03/11/ExtensionsRoundup.md

```diff
@@ -0,0 +1,80 @@
+---
+Order: 4
+TOCTitle: Extensions Roundup
+PageTitle: Visual Studio Code Extensions
+MetaDescription: New, useful, and interesting extensions in Visual Studio Code. 
+Date: 2016-03-11
+ShortDescription: New, useful, and interesting extensions in Visual Studio Code.
+Author: Wade Anderson
+---
+
+# VS Code Extensions
+
+Three months have passed since we open sourced VS Code and introduced the [extensions API](http://code.visualstudio.com/docs/extensionAPI/vscode-api). 
```

Comment:
maybe add "in November 2015" or "at Connect(); 2015" so this topic will age well?